charts/telemetry-collector/values.yaml

```
@@ -0,0 +1,167 @@
+global: {}
+
+nameOverride: ""
+fullnameOverride: ""
+
+replicaCount: 1
+
+bootstrap:
+  create: true
+  name: ""
+  kafka:
+    enabled: true
+    hostname: ""
+    port: 0
+  nats:
+    enabled: true
+    hostname: ""
+    port: 0
+
+image:
+  repository: accelleran/telemetry-collector
+  pullPolicy: IfNotPresent
+  tag: "0.9.0"
+
+imagePullSecrets:
+  - name: accelleran-secret
+
+accelleranLicense:
+  enabled: false
+  secretName: "accelleran-license"
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext:
+  capabilities:
+    drop:
+      - ALL
+  # readOnlyRootFilesystem: true
+  runAsNonRoot: true
+  runAsUser: 10000
+
+resources: {}
+  # limits:
+  #   cpu: 100m
+  #   memory: 128Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 128Mi
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+service:
+  enabled: true
+  name: ""
+  type: NodePort
```

Let the drax chart set this to `NodePort` later (where it probably will also define the nodeport itself). That way this chart remains more reusable as a building block let's say:

```suggestion
  type: ClusterIP
```

---

OK